docs/docs-source/docs/modules/get-started/pages/installing-cloudflow.adoc

```diff
@@ -0,0 +1,361 @@
+
+  helm upgrade nfs-server-provisioner stable/nfs-server-provisioner \
+    --install \
+    --set storageClass.provisionerName=cloudflow-nfs \
+    --namespace cloudflow
+
+The result of the installation, the NFS Server provisioner pod running and the new storage class.
+
+----
+$ kubectl get pods -n cloudflow
+NAME                       READY   STATUS    RESTARTS   AGE
+nfs-server-provisioner-0   1/1     Running   0          25s
+
+$ kubectl get sc
+NAME                 PROVISIONER            AGE
+nfs                  cloudflow-nfs          29s
+standard (default)   kubernetes.io/gce-pd   2m57s
+----
+
+NOTE: When installing Cloudflow, we will use the name `nfs` to indicate that Cloudflow should use the NFS storage class.
+
+== Installing Cloudflow
+
+In this guide, we will use Helm to install Cloudflow.
+
+The first step is to create the namespace we want to install Cloudflow into:
+
+  kubectl create ns cloudflow
+
+IMPORTANT: Many subsequent commands will assume that the namespace is `cloudflow`.
+
+First, we add the Cloudflow Helm repository:
+
+  helm repo add cloudflow-helm-charts https://lightbend.github.io/cloudflow-helm-charts/
+
+The `repo update` command ensures Helm has an up-to-date index of all Helm charts.
+
+  helm repo update
+
+Now we install Cloudflow, but before you press enter, let's go over the custom parameter we are supplying with the command:
+
+  cloudflow_operator.persistentStorageClass=nfs
+
+The `persistentStorageClass` parameter sets the storage class to use with Spark and Flink applications. This storage class, as previously mentioned, has to be of the type `ReadWriteMany`. In our example, we are using the `nfs` storage class, which we have installed in our cluster using.
+
+  cloudflow_operator.kafkaBootstrapservers=cloudflow-strimzi-kafka-bootstrap.cloudflow:9092
+
+The `kafkaBootstrapservers` parameter sets the address and port of the Kafka cluster that Cloudflow will use. In this example, we have used the address of a Strimzi created Kafka cluster located in the `cloudflow` Kubernetes namespace.
+
+Installing Cloudflow using the Helm chart.
+
+  helm install cloudflow cloudflow-helm-charts/cloudflow --namespace cloudflow --set \
+  strimzi.enabled=true --set cloudflow_operator.persistentStorageClass=nfs --set cloudflow_operator.kafkaBootstrapservers=cloudflow-strimzi-kafka-bootstrap.cloudflow:9092
+
+Check the status of the install process using `kubectl`. When the Cloudflow operator pod is in `Running` status, the installation is complete.
+
+----
+$ kubectl get pods -n cloudflow
+NAME                                                READY   STATUS    RESTARTS   AGE
+cloudflow-operator-6b7d7cbdfc-xb6w5                 1/1     Running   0          10s
+----
+
+You can now deploy an Akka based Cloudflow application into the cluster. More on this in the development section of the documentation.
+
+== Installing the Spark operator
```

Comment:
This section might be worth a separate page, just as Strimzi has its own.

Reply:
Good point. Will do.